Lusine_Shahbazyan/web/paginationWithFunction/script.js

```diff
@@ -0,0 +1,105 @@
+class pagination
+{
+    firstIndex;
+    lastIndex;
+
+    constructor(array, size)
+    {
+        this.arrayPage = array;
+        this.pageSize = size;
+        let div = document.getElementById("div2");
+
+        for(let i = 0; i < pageSize; i++)
+        {
+            let li1 = document.createElement("li");
+            li1.classList.add("li1ID");
+            li1.style.cursor = "pointer"
+            li1.innerHTML = i + 1;
+            li1.addEventListener("click", (event) => {
+                let res= this.goToPage(event.target.innerText)
+                this.printIntoHtml(res);
+            })
+            div.append(li1)
+        }
+        document.getElementById("next").addEventListener("click", () => {
+            let res= this.nextPage()
+            this.printIntoHtml(res);
+        })
+        document.getElementById("prev").addEventListener("click", () => {
+            let res= this.prevPage()
+            this.printIntoHtml(res);
+        })
+    }
+    printIntoHtml(value)
+    {
+        let var1 = document.getElementById("items");
+        var1.innerHTML = ""
+        for(let i = 0; i < value.length; i++)
+        {
+            let li2 = document.createElement("li");
+            li2.classList.add("li2ID");
+            li2.style.backgroundColor = "#d672bd";
+            li2.style.height = "1px"
+            li2.style.display = "block"
+            li2.style.border = "1px solid blue"
+            li2.style.borderRadius = "25px"
+            li2.innerHTML = value[i];
+            var1.appendChild(li2);
+        }
+    }
+    print(first, last)
```

Comment:
Please rename this function. It doesn't print anything so the name shouldn't be **print**.

Reply:
I have renamed print function.